juriscraper/pacer/email.py

```diff
@@ -202,6 +204,40 @@ def _get_description_plain(self):
             return clean_string(description)
         return None
 
+    def _contains_attachments(self) -> bool:
+        """Determines if the html notification contains attached documents.
+
+        :returns: True if it contains otherwise False.
+        """
+
+        path = '//p[contains(., "The following document(s) are associated with this transaction:")]/following::'
+        document_nodes = self.tree.xpath(
+            f'{path}strong[contains(., "Document description:")]'
+        )
+        if len(document_nodes) <= 1:
+            return False
+        return True
+
+    def _contains_attachments_plain(self) -> bool:
+        """Determines if the plain/txt notification contains attached documents.
+
+        :returns: True if it contains otherwise False.
+        """
+
+        mail_body = self.tree.text_content()
+        regex = r"^.*?The following document\(s\) are associated with this transaction:(.*?)$"
+
+        find_attachments = re.findall(regex, mail_body, re.DOTALL)
+        associated_documents = 0
+        if find_attachments:
+            splitlines = find_attachments[0].splitlines()
+            for index_line in range(len(splitlines)):
+                if "Document description:" in splitlines[index_line]:
+                    associated_documents = +1
+        if associated_documents <= 1:
```

Comment:
Nit: I'd do `== 0` instead of `<=1`

Reply:
I used <=1 because a notification might not contain any document, one or more than one. So that with <=1 we cover the case where there isn't any document in the notification or just one document (Main document).